scripts/import-spreadsheet.js

```diff
@@ -122,13 +121,19 @@ async function importNodes(nodes) {
         new Date(node.installDate),
         node.abandonDate ? new Date(node.abandonDate) : null,
         buildingsByNodeAddress[node.address].id,
-        membersMap[node.memberEmail.toLowerCase()].id,
       ];
     }
   );
   await performQuery(
     `ALTER SEQUENCE nodes_id_seq RESTART WITH ${maxNodeId + 1}`
   );
+
+  await insertBulk(
+    "memberships",
+    ["node_id", "member_id"],
```

Comment:
We should be able to dedupe nodes now -- we can bucket them by lat/lng and attach the members to the lowest node id. I'm happy to add this in a later pr but feel free to do it yourself if you'd like.

Reply:
Let's do this in a later PR. I don't understand the scope of the thing you're proposing yet.